bindings/python/tests/tests.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+
+from analiticcl import VariantModel, Weights, SearchParameters
+from icecream import ic
+
+LEXICON_AMPHIBIANS = 'tests/amphibians.tsv'
+LEXICON_REPTILES = 'tests/reptiles.tsv'
+
+
+class AnaliticclPythonBindingTests(unittest.TestCase):
+
+    def test_find_all_matches_with_multiple_lexicons(self):
+        model = VariantModel("tests/simple.alphabet.tsv", Weights(), debug=True)
+        model.read_lexicon(LEXICON_AMPHIBIANS)
+        model.read_lexicon(LEXICON_REPTILES)
+        model.build()
+        results = model.find_all_matches("Salamander lizard frog snake toad",
+                                         SearchParameters(max_edit_distance=3, max_ngram=1))
+        ic(results)
+
+        assert len(results) == 5
+        assert_result(results[0], 'Salamander', LEXICON_AMPHIBIANS, 'salamander')
+        assert_result(results[1], 'lizard', LEXICON_REPTILES)
+        assert_result(results[2], 'frog', LEXICON_AMPHIBIANS)
+        assert_result(results[3], 'snake', LEXICON_REPTILES)
+        assert_result(results[4], 'toad', LEXICON_AMPHIBIANS)
+
+
+def assert_result(result, orig_term, lexicon, lex_term=None):
```

Comment:
Je kan denk ik beter de assertIn() methode op unit.testcase gebruiken dan zelf een functie definiëren zoals hier, er zijn een aantal assert* methoden beschikbaar die out-of-the-box al voor zo'n soort output zorgen; zie https://docs.python.org/3.8/library/unittest.html#unittest.TestCase.assertEqual